casper/src/main/scala/coop/rchain/casper/genesis/contracts/ConfigRegistry.scala

```diff
@@ -0,0 +1,64 @@
+package coop.rchain.casper.genesis.contracts
+
+import coop.rchain.models.NormalizerEnv
+import coop.rchain.rholang.build.CompiledRholangTemplate
+import coop.rchain.shared.Base16
+
+final case class ConfigRegistry(
```

Comment:
This case class should be used in https://github.com/rchain/rchain/blob/dev/casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala#L54
like `ProofOfStake`.